components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -929,6 +939,10 @@ def create_link(parent_type, parent_id, child_type, child_id):
                     # return remaining links in same format as json above
                     # e.g. {"dataset":{"10":{"image":[1,2,3]}}}
                     for rl in remainingLinks:
+                        if not rl.loaded:
```

Comment:
Is this still a potential issue? If so it should be resolved, otherwise the proper logging should be put in place or an error condition raised.